### DISCUSSION TITLE
Can OsmAnd simply use a picture to as a map?

### DISCUSSION THREAD

**Discussion by cnchip** [osmandapp/OsmAnd#16236] [General]
This function is similar to Locus's Image Map Calibration:
[https://docs.locusmap.eu/doku.php?id=manual:user_guide:maps_tools:calibrator](https://docs.locusmap.eu/doku.php?id=manual:user_guide:maps_tools:calibrator)
![图片](https://user-images.githubusercontent.com/8869425/212476159-37669437-c1fc-423b-b352-9fabdaf0ce51.png)

Locus is an excellent software, however OsmAnd seems to be more professional on the map, so I actually also want to know the difference between them.

Many thanks.

**Comment by mariush444** [comment]
if you have already calibrated map then you can use https://github.com/mariush444/raster2osmand (python3)

If it is just picture you can calibrated in QGIS then save as mbtiles or sqlite.

But this is workaround and not solution.

**Reply by cnchip** [reply]
Thank you for your reply. Now I know!
Look forward to seeing solutions in the future. :)
Thanks.
